### ISSUE TITLE
Otimizar processamento

### ISSUE THREAD

**Issue by matholiveira91** [Jul10l1r4/D4N155#11]
O módulo **genarator.py** que é responsável por processar as string e gerar as wordlists está com gargalo de processamento mesmo utilizando Threads, talvez uma solução seja espcializar as Threads de forma a poupar processamento para os outros modulos. O módulo permutations também apresentam similar problema

**Comment by Jul10l1r4** [comment]
Sim, vdd... Está em nosso alvo a sua ideia de usar o processamento da GPU :metal:
